integration-tests/src/test/resources/compose/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       - target: 8080
         mode: host
     environment:
-      - CATALINA_OPTS=-Xdebug -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8000
+      - CATALINA_OPTS=-Xdebug -agentlib:jdwp=transport=dt_socket,server=y,suspend=n
```

Comment:
I removed this port. I hope this is the one you're talking about @Lars since you said remove it in docker-compose.yml. I'm telling you this since there was also a port ("Published:8000") in docker.compose.solr.yml.

Reply:
It's the published 8000 one. This just enables a debugport inside the container, which does not cause any issues.
I'm not `@Lars` by the way.